server-common/src/main/java/org/apache/gravitino/server/authorization/jcasbin/JcasbinAuthorizer.java

```diff
@@ -64,6 +69,9 @@ public class JcasbinAuthorizer implements GravitinoAuthorizer {
    */
   private Set<Long> loadedRoles = ConcurrentHashMap.newKeySet();
 
+  private static final Map<NameIdentifier, List<String>> ownerPolicyCache =
```

Comment:
Could we use the entity store unified cache?

Reply:
The purpose of this cache is to store owner policies previously loaded from Jcasbin, enabling their proper removal when the owner changes. However, it seems somewhat inappropriate to place this cache within the EntityStore.